src/engine/Runtime.v3

```diff
@@ -350,7 +350,19 @@ component Runtime {
 		stack.push(Value.Ref(new_cont));
 		return null;
 	}
-
+	// Finds a handler for {tag} by unwinding the stack-switching chain and return
+	// a freshly created continuation with {Continuation.bottom} pointing to the
+	// direct child of the handler stack.
+	def unwindStackChain(stack: WasmStack, instance: Instance, tag: u31, unwind_type: SuspensionType) -> Continuation {
```

Comment:
Similarly here, if `findSuspensionHandler` doesn't take the suspension type anymore, then this method won't need it.